tests/other_test_context.go

```diff
@@ -0,0 +1,113 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package tests
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"time"
+
+	"github.com/onsi/ginkgo/v2/formatter"
+	"github.com/stretchr/testify/require"
+
+	"github.com/ava-labs/avalanchego/wallet/subnet/primary/common"
+)
+
+const failNowMessage = "OtherTestContext.FailNow called"
+
+// TODO(marun) Consider a more descriptive name for this type
+type OtherTestContext struct {
```

Comment:
optional - seems like renaming this would be worthwhile in this PR

Reply:
`DefaultTestContext` ?

Reply:
Also not a huge fan of `Other`... `Simple`? `Fallback`? `Default`?